src/unified_ai_api.py

```
@@ -1713,11 +1707,12 @@ async def batch_transcribe_voice(
                     Path(temp_file_path).unlink(missing_ok=True)
 
             except Exception as exc:
+                logger.error(f"Batch transcription failed for file {i}: {str(exc)}", exc_info=True)
```

Ruff (0.12.2)

Use explicit conversion flag

Replace with conversion flag

(RUF010)